srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 7 nodes, setting nnodes to 1
2160
2160
2160
2160
2160
2160
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
2160
2160
2160
2160
2160
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: error: sh03-01n32: task 0: Exited with exit code 1
srun: Terminating job step 9431381.12
2160
2160
srun: error: sh03-01n32: task 0: Exited with exit code 1
srun: error: sh03-01n32: task 0: Exited with exit code 1
srun: error: sh03-01n32: task 0: Exited with exit code 1
srun: error: sh03-01n32: task 0: Exited with exit code 1
srun: error: sh03-01n32: task 0: Exited with exit code 1
srun: Terminating job step 9431381.8
srun: Terminating job step 9431381.13
srun: Terminating job step 9431381.10
srun: Terminating job step 9431381.9
srun: Terminating job step 9431381.11
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
2160
2160
2160
2160
2160
2160
srun: error: sh03-01n31: task 0: Exited with exit code 1
srun: Terminating job step 9431381.4
srun: error: sh03-01n31: task 0: Exited with exit code 1
srun: error: sh03-01n31: task 0: Exited with exit code 1
srun: Terminating job step 9431381.7
srun: error: sh03-01n31: task 0: Exited with exit code 1
srun: error: sh03-01n31: task 0: Exited with exit code 1
srun: Terminating job step 9431381.6
srun: Terminating job step 9431381.5
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: Terminating job step 9431381.3
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: error: sh03-01n29: task 0: Exited with exit code 1
2160
srun: Terminating job step 9431381.0
srun: error: sh03-01n29: task 0: Exited with exit code 1
srun: Terminating job step 9431381.2
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: error: sh03-01n29: task 0: Exited with exit code 1
srun: Terminating job step 9431381.1
srun: error: sh03-01n33: task 0: Exited with exit code 1
srun: Terminating job step 9431381.14
srun: error: sh03-01n33: task 0: Exited with exit code 1
srun: error: sh03-01n33: task 0: Exited with exit code 1
srun: error: sh03-01n33: task 0: Exited with exit code 1
srun: error: sh03-01n33: task 0: Exited with exit code 1
srun: error: sh03-01n33: task 0: Exited with exit code 1
srun: Terminating job step 9431381.15
srun: Terminating job step 9431381.17
srun: Terminating job step 9431381.19
srun: Terminating job step 9431381.18
srun: Terminating job step 9431381.16
2160
2160
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: error: sh03-01n35: task 0: Exited with exit code 1
srun: Terminating job step 9431381.24
srun: error: sh03-01n35: task 0: Exited with exit code 1
srun: Terminating job step 9431381.23
2160
2160
2160
Traceback (most recent call last):
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
    main()
  File "run_linearization_heuristic.py", line 99, in main
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
    delta = all_instances[instance_index][0]
IndexError: list index out of range
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: error: sh03-01n34: task 0: Exited with exit code 1
srun: error: sh03-01n34: task 0: Exited with exit code 1
srun: error: sh03-01n34: task 0: Exited with exit code 1
srun: Terminating job step 9431381.21
srun: Terminating job step 9431381.22
srun: Terminating job step 9431381.20
2160
2160
2160
2160
2160
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: error: sh03-01n38: task 0: Exited with exit code 1
srun: Terminating job step 9431381.28
srun: error: sh03-01n38: task 0: Exited with exit code 1
srun: error: sh03-01n38: task 0: Exited with exit code 1
srun: error: sh03-01n38: task 0: Exited with exit code 1
srun: error: sh03-01n38: task 0: Exited with exit code 1
srun: Terminating job step 9431381.27
srun: Terminating job step 9431381.26
srun: Terminating job step 9431381.25
srun: Terminating job step 9431381.29
